README.md

```diff
@@ -1 +1,150 @@
-# template
\ No newline at end of file
+<style>
+	.settings { border-collapse: collapse; }
+	.settings th, .settings td { padding: 5px; border: solid 1px #777; vertical-align: top }
+	.settings th { background-color: lightblue; text-transform: uppercase; font-weight: normal; text-align: left}
+	.code {font-family:Courier New,Courier,monospace; background-color:#dddddd; font-size: 14px }
+</style>
+
+<h2>pw-parking-permit</h2>
+
+<p>This is a template repository that can be used when developing processes in Camunda together with SpringBoot.</p>
```

Comment:
Change content of readme to better describe what this process-worker-service does? Some stuff can be kept (such as description on how to start and update a process instance), but most of it describes the template and not this service.

Reply:
Will do this. I thought it was not in scope for this task but it should be done anyway.